src/181810.c

```diff
@@ -554,32 +562,32 @@ void save_tattle_flags(s32 actorType) {
         i++;
     }
 
-    gb = get_global_byte((actorType / 8) + 0x16D);
+    gb = get_global_byte(EVT_INDEX_OF_GAME_BYTE(GB_Tattles_00) + actorType / 8);
     gb |= 1 << (actorType % 8);
-    set_global_byte((actorType / 8) + 0x16D, gb);
+    set_global_byte(EVT_INDEX_OF_GAME_BYTE(GB_Tattles_00) + actorType / 8, gb);
     battleStatus->tattleFlags[actorType / 8] |= gb;
 }
 
 void load_tattle_flags(s32 actorType) {
     BattleStatus* battleStatus = &gBattleStatus;
-    u8* list;
+    u8* types;
     s32 gb;
     s32 i = 0;
     s32 j;
     s32 k;
 
     while (TRUE) {
-        list = ActorTypesLists[i];
-        if (list == NULL) {
+        types = ActorTypesLists[i];
+        if (types == NULL) {
             break;
         }
 
-        for (j = 0; list[j] != 0xFF; j++) {
-            if (actorType == list[j]) {
-                for (k = 0; list[k] != 0xFF; k++) {
-                    actorType = list[k];
+        for (j = 0; types[j] != 0xFF; j++) {
+            if (actorType == types[j]) {
+                for (k = 0; types[k] != 0xFF; k++) {
+                    actorType = types[k];
 
-                    gb = get_global_byte((actorType / 8) + 0x16D); // GameByte(0x16D) is GB_Tattles_00 (first tattle)
+                    gb = get_global_byte((actorType / 8) + EVT_INDEX_OF_GAME_BYTE(GB_Tattles_00));
```

Comment:
Maybe also swap these around, so it becomes `EVT_INDEX_OF_GAME_BYTE(GB_Tattles_00) + actorType / 8`? Or add a macro for getting this byte, considering how often this is used?

Reply:
mhm, like `EVT_TATTLE_GAME_BYTE(actorType)`

Reply:
Ah, I missed this one. Good catch.
Thought about making a macro, but decided against it because this actorType -> flag calculation is done in a couple different ways in this file and it doesn't make much sense to macro-ify some of them but not others. 
Would be nice if we could match with something like
`s32 flagByte = actorType / 8;`
`s32 flagIdx = actorType % 8;`
but unfortunately, it does not match that way.